crates/wit-component/src/printing.rs

```diff
@@ -82,6 +82,14 @@ impl WitPrinter {
                 self.output.push_str("interface ");
                 self.print_name(name);
                 self.output.push_str(" {\n");
+                let nested = &resolve.interfaces[*id].nested;
+                for item in nested {
+                    self.print_stability(&item.1.stability);
+                    self.print_docs(&item.1.docs);
+                    self.output.push_str("nest ");
+                    self.print_name(item.0);
```

Comment:
I don't think that this is quite right because this should print `nest a:b/c;` or similar so I think an interface id needs to be printed here rather than just the string name

Reply:
This got moved around a bit, but I can update once the above comment is resolved re: how nested ids/interface names are captured.  FWIW though, the string used is the interface id at the moment, but I can see how grabbing the interface id would be a bit less clunky.